example.yml

```diff
@@ -209,3 +209,18 @@ modules:
       transport_protocol: "tcp" # defaults to "udp"
       preferred_ip_protocol: "ip4" # defaults to "ip6"
       query_name: "www.prometheus.io"
+  websocket_example:
+    prober: websocket
+    websocket:
+      http_config:
+        basic_auth:
+          username: "user"
+          password: "password"
+        bearer_token: "secret_token" 
```

Comment:
does it make sense to configure `bearer_token` and `basic_auth` both? usually you would use one or the other. if you want to show an example of both config, please create two examples instead of putting both in one example.